tests/test_other.py

```diff
@@ -10004,6 +10004,12 @@ def test_old_makeDynCall_syntax(self):
     err = self.run_process([EMCC, test_file('test_old_dyncall_format.c'), '--js-library', test_file('library_test_old_dyncall_format.js')], stderr=PIPE).stderr
     self.assertContained('syntax for makeDynCall has changed', err)
 
+  # Tests that dynCalls are produced in Closure-safe way in DYNCALLS mode when no actual dynCalls are used
+  def test_closure_no_dynCalls_generated(self):
+    self.run_process([EMCC, test_file('hello_world.c'), '--closure=1'])
+    self.run_process([EMCC, test_file('hello_world.c'), '-s', 'ASYNCIFY', '--closure=1'])
+    self.run_process([EMCC, test_file('hello_world.c'), '-s', 'ASYNCIFY', '-s', 'WASM_BIGINT', '--closure=1'])
+
```

Comment:
This could be
```python
  @parameterized({
    'plain': [[]],
    'asyncify': [['-sASYNCIFY']],
    'asyncify-bigint': [['-sASYNCIFY', '-sWASM_BIGINT']]})
  def test_closure_no_dynCalls_generated(self, args):
    self.run_process([EMCC, test_file('hello_world.c'), '--closure=1'] + args)